site.yml.sample

```diff
@@ -68,6 +68,27 @@
         - ansible_distribution == 'Fedora'
         - ansible_distribution_major_version|int >= 23
 
+- hosts:
```

Comment:
can we make line 25 https://github.com/ceph/ceph-ansible/pull/2555/files#diff-5e11e06bb360175675e3ae901c5d08dbR25 a pre task and then run the validation right after instead of re-declaring a `hosts` block again?